pkg/vm/vm.go

```diff
@@ -236,12 +236,84 @@ func (v *VM) Step() {
 	}
 }
 
+// StepInto behaves the same as “step over” in case if the line does not contain a function it otherwise
+// the debugger will enter the called function and continue line-by-line debugging there.
+func (v *VM) StepInto() {
+	ctx := v.Context()
+
+	if ctx == nil {
+		v.state |= haltState
+	}
+
+	if v.HasStopped() {
+		return
+	}
+
+	if ctx != nil && ctx.prog != nil {
+		var op Instruction
+		if ctx.ip >= len(ctx.prog) {
+			op = RET
```

Comment:
I think this logic is already present in the `Next` itself.

Reply:
true, just calling next in that case and having same fmt logs as in step